spec/controllers/master_files_controller_spec.rb

```diff
@@ -252,6 +249,10 @@ class << file
 
     context 'with fedora 3 pid' do
       let!(:master_file) { FactoryBot.create(:master_file, identifier: [fedora3_pid]) }
+      let(:media_object) do
+        instance_double('MediaObject', title: 'Media Object', id: 'avalon:1234',
+                                       ordered_master_files: [master_file], master_file_ids: [master_file.id])
+      end
```

Comment:
I'm not seeing where this is used in this example.  Maybe it could be removed?

Reply:
It is necessary for the check at ln 259. There may be a way to refactor it to not need the instance double but everything I've tried seems to cause an infinite loop.